upgrade/money-class.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Is this blank line intentional?

Reply:
e36554e78789d500707b3ca6c3ccbe4284107b71